src/react/chat-status-response.ts

```diff
@@ -0,0 +1,18 @@
+import { ConnectionStatus, ErrorInfo, RoomStatus } from '@ably/chat';
+
+/**
+ * Common status variables for chat features. Most hooks in this library implement this interface.
```

Comment:
To make typedocs work, you will need to adjust the paths typedocs uses to find the exported types. My open PR does this if you want to copy it :) Else it should be merged soon and a rebase will fix this!